build.gradle

```diff
@@ -22,38 +22,38 @@ dependencies {
     implementation 'org.springframework.boot:spring-boot-starter-oauth2-client'
     implementation("org.springframework.boot:spring-boot-starter-webflux")
 
-    implementation 'com.unboundid:unboundid-ldapsdk:5.1.4'
+    implementation 'com.unboundid:unboundid-ldapsdk:6.0.2'
     implementation("io.springfox:springfox-boot-starter:3.0.0")
-    implementation 'org.flywaydb:flyway-core:7.9.0'
-    implementation 'com.zaxxer:HikariCP:4.0.3'
-    implementation 'commons-io:commons-io:2.8.0'
+    implementation 'org.flywaydb:flyway-core:8.0.5'
+    implementation 'com.zaxxer:HikariCP:5.0.0'
+    implementation 'commons-io:commons-io:2.11.0'
     implementation 'org.apache.commons:commons-lang3:3.12.0'
     implementation 'org.apache.commons:commons-text:1.9'
     implementation 'io.swagger:swagger-core:1.6.2'
-    implementation 'com.oracle.database.jdbc:ojdbc10:19.10.0.0'
-    implementation 'io.vavr:vavr:0.10.3'
+    implementation 'com.oracle.database.jdbc:ojdbc10:19.12.0.0'
+    implementation 'io.vavr:vavr:0.10.4'
     implementation 'com.github.java-json-tools:json-patch:1.13'
 
     // needed for record serialisation
-    implementation 'com.fasterxml.jackson.core:jackson-core:2.12.4'
-    implementation 'com.fasterxml.jackson.core:jackson-databind:2.12.4'
-    implementation 'com.fasterxml.jackson.core:jackson-annotations:2.12.4'
+    implementation 'com.fasterxml.jackson.core:jackson-core:2.12.5'
+    implementation 'com.fasterxml.jackson.core:jackson-databind:2.12.5'
+    implementation 'com.fasterxml.jackson.core:jackson-annotations:2.12.5'
 
-    annotationProcessor 'org.projectlombok:lombok:1.18.20'
+    annotationProcessor 'org.projectlombok:lombok:1.18.22'
     annotationProcessor 'org.springframework.boot:spring-boot-configuration-processor'
-    compileOnly 'org.projectlombok:lombok:1.18.20'
+    compileOnly 'org.projectlombok:lombok:1.18.22'
 
     runtimeOnly 'com.h2database:h2:1.4.200'
     testRuntimeOnly 'com.h2database:h2:1.4.200'
-    testAnnotationProcessor 'org.projectlombok:lombok:1.18.20'
-    testCompileOnly 'org.projectlombok:lombok:1.18.20'
+    testAnnotationProcessor 'org.projectlombok:lombok:1.18.22'
+    testCompileOnly 'org.projectlombok:lombok:1.18.22'
 
     testImplementation 'org.codehaus.groovy.modules.http-builder:http-builder:0.7.1'
-    testImplementation ('io.rest-assured:rest-assured:3.3.0'){
+    testImplementation ('io.rest-assured:rest-assured:4.4.0'){
```

Comment:
if the tests work with this that fantastic, for some reason I though the tests would break on v4

Reply:
I'm sure I tried it previously and they didn't work, but they all appear to have passed :grin: